AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -50,6 +50,8 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101941
+d.dmkt-sp.jp###Anniversary
```

Comment:
Have you reproduced it? I didn't see it on my end.

Reply:
It was there when I sent this pull request.
But now it has been replaced by a different banner.

`d.dmkt-sp.jp##.fltBnrBlock`

Reply:
> Have you reproduced it? I didn't see it on my end.

It was there when I sent this pull request. #102523
But now it has been replaced by another one. → `d.dmkt-sp.jp##.fltBnrBlock`
I have not reproduced it. → `d.dmkt-sp.jp###Anniversary`



![](https://files.catbox.moe/hw8t0r.png)

Reply:
Well, remove `d.dmkt-sp.jp###Anniversary` and add `d.dmkt-sp.jp##.fltBnrBlock` to the `AdGuard Annoyances - popups`.

Reply:
ok.done